src/server/handlers/github.js

```diff
@@ -0,0 +1,96 @@
+import { conf } from '../helpers/config';
+import request from 'request';
+
+const GITHUB_USERNAME = conf.get('GITHUB_USERNAME');
+const GITHUB_PASSWORD = conf.get('GITHUB_PASSWORD');
+const GITHUB_API_ENDPOINT = conf.get('GITHUB_API_ENDPOINT')
+const HTTP_PROXY = conf.get('HTTP_PROXY');
+
+const RESPONSE_NOT_FOUND = {
+  status: "error",
+  payload: {
+    message: "github-repository-not-found"
+  }
+};
+
+const RESPONSE_AUTHENTICATION_FAILED = {
+  status: "error",
+  payload: {
+    message: "github-authentication-failed"
+  }
+};
+
+const RESPONSE_OTHER = {
+  status: "error",
+  payload: {
+    message: "github-error-other"
+  }
+}
+
+const RESPONSE_ALREADY_CREATED = {
+  status: "error",
+  payload: {
+    message: "github-already-created"
+  }
+}
+
+const RESPONSE_CREATED = {
+  status: "success",
+  payload: {
+    message: "github-webhook-created"
+  }
+}
```

Comment:
I don't have any strong opinions about the shape of the response. This proposal seems quite fine to me.

One comment - it may be useful to provide both the 'code' and textual message. In this case, something like:

```json
payload: {
   "code": "github-webhook-created",
   "message": "GitHub webhook successfully created."
}
```